daemon/web/index.html

```diff
@@ -4,7 +4,8 @@
 <head>
   <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Rubik" rel="stylesheet">
-  <title>Inertia Web</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
```

Comment:
I believe fontawesome was already added into the index.html by someone else. Do you have the latest master branch?